### PR TITLE
Troubleshooting section added to README.md

### PR DESCRIPTION
**Pull request by stranger80**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,39 @@ If you're looking to utilize the capabilities of Starknet contracts within the R
...
 
+### Troubleshooting
+
+Most issues with Starknet plugin or Remix itself are caused by connectivity problems (also resulting from restricted networks, web-proxies blocking certain content, etc.) or browser plugin interference. 
+- A user should first attempt to disable any browser components & addons which may impact the connectivity or Javascript execution. 
+- Some networks may restrict connectivity to certain sites or domains. Using a VPN conneciton may resolve problems observed on restricted networks.
+
+More specific potential error causes are also described in detail below. 
+
+1. **Problems when searching plugins in Remix UI**
```
Comment (swapnilraj) [inline README.md:50]: can we make these dropdown 

https://gist.github.com/citrusui/07978f14b11adada364ff901e27c7f61

Comment (stranger80) [reply README.md]: Dropdowns added

File: README.md [hunk 2]
```diff
@@ -39,6 +39,54 @@ If you're looking to utilize the capabilities of Starknet contracts within the R
...
+</details>
+
+<details>
+<summary><strong>2. Problems when launching Starknet plugin in Remix UI</strong></summary> 
+<br/>  
+Starknet plugin launch issues may be caused by connectivity errors or plugin components being unavailable. This can be tested from web browser, as indicated below:
+
+- https://cairo-remix-test.nethermind.io should respond with blank page (advanced: viewing page source will reveal a React component entry HTML markup) - no errors should be reported by the browser
+- https://cairo-compile-remix-test.nethermind.io/health should respond with `OK`
+- https://starknet-devnet-dev.nethermind.io/feeder_gateway/get_transaction_status?transactionHash=0xbadfad should respond with `{"tx_status":"NOT_RECEIVED"}`
```
Comment (prix0007) [inline README.md:70]: maybe here add an example from the rpc endpoint also, since feeder gateway will be deprecated soon (in few months),

**Review by prix0007 — APPROVED** [review]
LGTM, just left a comment

File: README.md (hunk 2 above)
Comment (stranger80) [reply README.md]: Actually I've replaced it with generic https://starknet-devnet-dev.nethermind.io/predeployed_accounts - whish is just enough to check devnet connectivity.

**Pull request merged**